toml/decoder.py

```diff
@@ -417,9 +445,9 @@ def _load_unicode_escapes(v, hexbytes, prefix):
         while i < hxblen:
             try:
                 if not hx[i].lower() in hexchars:
-                    raise IndexError("This is a hack")
-            except IndexError:
-                raise TomlDecodeError("Invalid escape sequence")
+                    raise ValueError()
+            except (IndexError, ValueError):
```

Comment:
I think this should still be an `IndexError`? Why the change to `ValueError`?

Reply:
I thought the index error was a hack to fall into this `except`, and that a `ValueError` was more appropriate for an invalid hex character.

Reply:
I suppose you are right. It doesn't really matter (to me) _which_ error type is used but I think the `except` should only accept the intentional error which gets raised for an invalid hex character.